PoshRSJob/PoshRSJob.psm1

```diff
@@ -48,7 +48,7 @@ Add-Type -TypeDefinition @"
             public string NewVarName;
         }
     }
-"@ -Language CSharpVersion3
+"@ -Language CSharp
```

Comment:
I thought I had this fixed before but it must have snuck back in during one of my updates. Thanks!